doc.go

```diff
@@ -22,8 +22,183 @@ instance:
         })
     }
 
+Assertions
+
+An assertion looks like this, where qt.Equals could be replaced by any
+available checker. If the assertion fails, the underlying Fatal method is
+called to describe the error and abort the test.
+
+    c.Assert(someValue, qt.Equals, wantValue)
+
+If you don’t want to abort on failure, use Check instead, which calls Error
+instead of Fatal:
+
+    c.Check(someValue, qt.Equals, wantValue)
+
 The library provides some base checkers like Equals, DeepEquals, Matches,
 ErrorMatches, IsNil and others. More can be added by implementing the Checker
-interface.
+interface. Here is a list of checkers already included in the package.
+
+Equals
+
+Equals is a Checker checking equality of two comparable values.
```

Comment:
Equals checks that two values are equal, as compared with Go's == operator.

?

Reply:
Done.